docs/en/glossary/glossary.asciidoc

```diff
@@ -662,8 +692,14 @@ Code is considered instrumented when it collects and reports this performance da
 //Source: Observability
 
 [[glossary-integration]] integration::
-Out-of-the-box configurations for common data sources to simplify the collection,
-parsing, and visualization of logs and metrics. Also known as a <<glossary-module,module>>.
+A bundle of assets for configuring the ingestion and visualization of data,
+or the protection of endpoints.
+Any number of integrations can be added to an <<glossary-agent-policy,{agent} policy>>.
```

Comment:
I wonder here if we should just stick with a simple definition and not talk about agent policies. I think it helps to future proof a definition if we avoid including extra details that aren't critical to the meaning of a term.

Reply:
An integration provides an easy way for Elastic to interact with external systems, whether its collecting data or protecting systems from security threats. Many provide out of the box assets to help you get started and get insights faster. Many can also be configured on {agent}s in a single click.